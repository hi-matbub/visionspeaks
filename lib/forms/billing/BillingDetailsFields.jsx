import React from 'react';
import Form from 'react-bootstrap/Form';

const BillingDetailsFields = () => (
  <Form.Row>
    <h2 className="mt-5">Billing Information</h2>

    <Form.Control
      className="mt-3"
      type="text"
      label="name"
      name="name"
      placeholder="Jane Doe"
      required
    />

    <Form.Control
      className="mt-3"
      type="text"
      label="email"
      name="email"
      placeholder="jane.doe@example.com"
      required
    />

    <Form.Control
      className="mt-3"
      type="text"
      label="address"
      name="address"
      placeholder="185 Berry St. Suite 550"
      required
    />

    <Form.Control
      className="mt-3"
      type="text"
      label="city"
      name="city"
      placeholder="San Francisco"
      required
    />

    <Form.Control
      className="mt-3"
      type="text"
      label="state"
      name="state"
      placeholder="CA"
      required
    />

    <Form.Control
      className="my-3"
      type="text"
      label="zip"
      name="zip"
      placeholder="94103"
      required
    />
  </Form.Row>
);

export default BillingDetailsFields;
