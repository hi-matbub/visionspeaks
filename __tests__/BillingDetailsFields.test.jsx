import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import BillingDetailsFields from '../lib/forms/billing/BillingDetailsFields';

it('BillingDetailsFields component renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<BillingDetailsFields />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('BillingDetailsFields component renders unchanged', () => {
  const tree = renderer.create(<BillingDetailsFields />).toJSON();
  expect(tree).toMatchSnapshot();
});
