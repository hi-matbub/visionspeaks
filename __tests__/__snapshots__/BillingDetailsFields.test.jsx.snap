// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BillingDetailsFields component renders unchanged 1`] = `
<div
  className="form-row"
>
  <h2
    className="mt-5"
  >
    Billing Information
  </h2>
  <input
    className="mt-3 form-control"
    label="name"
    name="name"
    placeholder="Jane Doe"
    required={true}
    type="text"
  />
  <input
    className="mt-3 form-control"
    label="email"
    name="email"
    placeholder="jane.doe@example.com"
    required={true}
    type="text"
  />
  <input
    className="mt-3 form-control"
    label="address"
    name="address"
    placeholder="185 Berry St. Suite 550"
    required={true}
    type="text"
  />
  <input
    className="mt-3 form-control"
    label="city"
    name="city"
    placeholder="San Francisco"
    required={true}
    type="text"
  />
  <input
    className="mt-3 form-control"
    label="state"
    name="state"
    placeholder="CA"
    required={true}
    type="text"
  />
  <input
    className="my-3 form-control"
    label="zip"
    name="zip"
    placeholder="94103"
    required={true}
    type="text"
  />
</div>
`;
